terraform/accounts/main/jenkins_instance_profile.tf

```diff
@@ -159,9 +159,28 @@ data "aws_iam_policy_document" "cloudwatch-policy" {
   }
 }
 
-resource "aws_iam_role_policy" "jenkins-cloudwatch" {
+resource "aws_iam_role_policy" "jenkins-cloudwatch-logs" {
   name = "JenkinsCloudWatchLogs"
   role = "${aws_iam_role.jenkins.id}"
 
-  policy = "${data.aws_iam_policy_document.cloudwatch-policy.json}"
+  policy = "${data.aws_iam_policy_document.cloudwatch-logs-policy.json}"
+}
+
+data "aws_iam_policy_document" "cloudwatch-metrics-policy" {
+  statement {
+    actions = [
+      "cloudwatch:PutMetricData",
+    ]
+
+    resources = [
+      "*",
```

Comment:
Does this mean 'everything'? Or are we waiting on a hardcoded value?

Reply:
Means everything, and according to the AWS interface has to be everything.